pkgs/games/bzflag/default.nix

```diff
@@ -15,6 +15,17 @@ stdenv.mkDerivation rec {
   buildInputs = [ curl SDL2 libGLU libGL glew ncurses c-ares ]
     ++ lib.optionals stdenv.isDarwin [ Carbon CoreServices ];
 
+  enableParallelBuilding = true;
+
+  postBuild = ''
+    make -C misc bzflag.desktop
```

Comment:
use `installTargets` instead.

Reply:
Can you show how this would work please?  My knowledge of autotools is limited.  I see this `bzflag.desktop` target and a related `install-desktopDATA` target in `misc/Makefile.am` and `misc/Makefile.in` respectively, but I don't see how to do `-C misc` to run them with `installTargets`.  `make misc` and `make dist` (via `installTargets`) from the top-level directory don't do the trick either (the former gives "make: Nothing to be done", the latter seems to expect a git repo).

Reply:
See https://github.com/NixOS/nixpkgs/blob/c83315ed5d7aa1fd203f1739d7cb4638816895ea/pkgs/applications/audio/mpg321/default.nix#L34

Reply:
Yes, I got how `installTargets` works, but after reading the Makefiles, I didn't see a way to build or install it.

Anyway, after coming back to this and looking at the two desktop files again, the `data/` one is actually better, plus it's automatically installed by the Makefile -- just into the wrong place.  PR updated to use that one instead.